src/pages/ItemList/ItemList.js

```diff
@@ -0,0 +1,66 @@
+import { useState, useEffect } from 'react';
+import './ItemList.scss';
+
+function ItemList() {
+  const [list, setList] = useState([]);
+
+  useEffect(() => {
+    fetch('http://10.58.0.75:8000/products')
+      .then(res => res.json())
+      .then(data => {
+        setList(data.results);
+      });
+  }, []);
+
+  return (
+    <div className="itemList">
+      <div className="itemListContent">
+        <ul className="itemListCotentGallery">
+          {list.map(content => {
+            return (
+              <li key={content.id} className="listGallery">
+                <div className="galleryItem">
+                  <div className="itemPhotoBox">
+                    <a href="#" className="itemPageMove">
+                      <img
+                        className="listImage"
+                        src={content.thumbnail_image}
+                        alt="listItem"
+                      />
+                    </a>
+                  </div>
+                  <div className="itemDiscountRate">
+                    {content.discount_rate === ''
+                      ? ''
+                      : `${content.discount_rate * 100}%`}
+                  </div>
```

Comment:
discount_rate가 없는 경우에는 아예 렌더링되지 않도록 하시는 게 더 좋을 것 같습니다.

```suggestion
                  {content.discount_rate && (
                    <div className="itemDiscountRate">
                      {content.discount_rate * 100}%
                    </div>
                  }
```